app/controllers/announcements_controller.rb

```diff
@@ -34,4 +36,28 @@ def index
   def scheduled_announcements
     Announcement.scheduled.order("scheduled_publication asc")
   end
+
+  def is_english_locale?
+    params[:locale].nil?
+  end
+
+  def redirect_to_news_and_communications
+    base_path = "#{Plek.new.website_root}/search/news-and-communications"
+    redirect_to("#{base_path}?#{news_and_communications_query_string}")
+  end
+
+  def news_and_communications_query_string
+    {
+      keywords: params['keywords'],
+      level_one_taxon: params['taxons'].try(:first),
+      level_two_taxon: params['subtaxons'].try(:first),
+      people: params['people'],
+      organisations: params['departments'],
+      world_locations: params['world_locations'],
+      public_timestamp: {
+        from: params['from_date'],
+        to: params['to_date']
+      }.compact.presence,
+    }.to_query
```

Comment:
Would be good to filter out any parameters that are empty, I tried going to `government/announcements` and it redirected me to `search/news-and-communications?keywords=&level_one_taxon=&level_two_taxon=&organisations=&people=&public_timestamp=&world_locations=` which is a bit ungainly

Reply:
Good idea - done!